modules/javafx.controls/src/test/java/test/javafx/scene/control/TableViewTest.java

```diff
@@ -5795,4 +5795,52 @@ public void testAnchorRemainsWhenAddingMoreItemsBelow() {
         assertEquals(1, anchor.getRow());
         assertEquals(column, anchor.getTableColumn());
     }
+
```

Comment:
please update the copyright year in TableViewTest as well, since it's been modified:

 * Copyright (c) 2010, 2022, Oracle and/or its affiliates. All rights reserved.